packages/ChangesReloaded-Tests.package/CRBrowserTest.class/instance/assertNoChangesAreLoadedWhenAllUnpicked.st

```diff
@@ -0,0 +1,6 @@
+assertions
+assertNoChangesAreLoadedWhenAllUnpicked
+	
+	self assert: (self changesBrowser changeListItems allSatisfy: [:each | each isInImage not]).
+	self changesBrowser loadPickedChanges.
+	self assert: (self changesBrowser changeListItems allSatisfy: [:each | each isInImage not]).
```

Comment:
Remove dot at the end

Reply:
this was still in my review no idea if this is fixed